### PR TITLE
fix: update arguments name

### PR DESCRIPTION
**Pull request by makram93**
Update the arguments name

### CODE REVIEW

File: executor.py [hunk 1]
```diff
@@ -59,19 +58,18 @@ def __init__(
...
         ).logger
-        self.max_num_training_points = max_num_training_points
         self.cluster_center = cluster_center
         self.iter = iter_steps
         self.metric = 'euclidean'
-        self.subspaces = num_subspaces
-        if num_codeword and num_codeword > 256:
+        self.subspaces = subspaces
+        if codewords and codewords > 256:
             self.logger.warning(
```
Comment (cristianmtr) [inline executor.py:66]: So this is a hard limit for this parameter?

**Review by cristianmtr — CHANGES_REQUESTED** [review]
Did no tests get affected by this?

File: executor.py (hunk 1 above)
Comment (makram93) [reply executor.py:66]: Yes, its from Rii library

**Comment by makram93** [comment]
> Did no tests get affected by this?

No, because I just replaced the arguments so that it is consistent

**Review by cristianmtr — APPROVED** [review]

**Comment by cristianmtr** [comment]
Ok cool.

**Pull request merged**